File: 5/Problem5.c
#include<stdio.h>

void MultipleDisplay(int iNo)
{
    int iCnt = 0;
    int iMult = 0;
    if(iNo < 0 )
    {
        printf("Ivalid Input\n");       // Filter
    }
    for(iCnt = 1 ; iCnt <= 5 ; iCnt++)
    {
        iMult = iNo * iCnt;
        printf("%d\n",iMult);
    }

}
    

int main()
{
    int iValue = 0;
    printf("Enter the number :");
    scanf("%d",&iValue);

    MultipleDisplay(iValue);

    return 0;
}

File: 17/problem5.java
import java.util.*;

class DigitX
{
    public void Table(int iNo)
    {
        int iCnt = 0,iMult = 0;
        for(iCnt = 1 ; iCnt <= 10; iCnt++)
        {
            iMult = iNo * iCnt;
            System.out.println(iNo+ "x" +iCnt +"=" +iMult);
        }
        
    }
}
class problem5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        DigitX dobj = new DigitX();

        int iValue = 0, iRet = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        dobj.Table(iValue);

    }
}